src/ocrd/processor/base.py

```diff
@@ -131,50 +175,51 @@ def __init__(
             for res in self.list_all_resources():
                 print(res)
             return
-        if resolve_resource or show_resource:
-            initLogging()
+        if resolve_resource:
             try:
-                res_fname = self.resolve_resource(resolve_resource or show_resource)
+                res = self.resolve_resource(resolve_resource)
+                print(res)
+            except ResourceNotFoundError as e:
+                log = getLogger('ocrd.processor.base')
```

Comment:
Could we find a way to define `self.logger` and use that consistently?

Reply:
I thought about it, and providing something standardised (esp. having predictable logger names like `'processor.' + self.__class__.__name__`) would be useful indeed.

But we should also make sure to differentiate between logging from the base class (`ocrd.processor.base`) and the processor implementation itself.

Reply:
True, and I don't see a way to do that cleanly without some `inspect` magic to know the inheriting class. But if possible, a standardized way would be appreciated (and I still have logging broken for non `ocrd.` loggers on my TODO list).

Reply:
I don't think it's as complicated as all that, bertsky/core#10